node/src/blockchain/blockchain_bridge.rs

```diff
@@ -295,24 +287,36 @@ impl BlockchainBridge {
         };
         let consuming_wallet_balances = {
             ConsumingWalletBalances {
-                gas_currency: gas_balance,
-                masq_tokens: token_balance,
+                gas_currency_wei: gas_balance,
+                masq_tokens_wei: token_balance,
             }
         };
+        let desired_gas_price_gwei = self
+            .persistent_config
+            .gas_price()
+            .map_err(|e| format!("Couldn't query the gas price: {:?}", e))?;
+        let estimated_gas_limit_per_transaction =
+            self.blockchain_interface.estimated_gas_limit_per_payable();
+        let this_stage_data = StageData::FinancialDetails {
```

Comment:
You thought `FinancialDetails` could be named better. How about `PreliminaryContext`?